nodejs/watchtower/integration.tests/subscribe.topic.test.js

```diff
@@ -0,0 +1,88 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const urlHandler = require('../../handlers/url.handler');
+const watchtowerHandler = require('../../handlers/watchtower.handler');
+const io = require('socket.io-client');
+let socket;
+let topicSubscriptionPayload;
+const topicTestnet = `3493291`;
```

Comment:
Is this going to fail after a testnet reset?

Reply:
Yes 🥲